src/displayapp/screens/WatchFaceDigital.h

```diff
@@ -85,7 +92,8 @@ namespace Pinetime {
                                              controllers.notificationManager,
                                              controllers.settingsController,
                                              controllers.heartRateController,
-                                             controllers.motionController);
+                                             controllers.motionController,
+                                             *controllers.weatherController);
```

Comment:
Why is the weathercontroller the only one that gets dereferenced? Just curious as I don't know the weather code quite well

Reply:
The reason this is like this is because the BLE components are initialised later by the NimbleController, so they can't be stored as references in the AppControllers struct created by DisplayApp. It would probably be good to refactor this, as it might clean up some things, but that's definitely outside the scope of this PR.